src/weapon/w_002.c

```diff
@@ -3,7 +3,57 @@
 #include "weapon_private.h"
 #include "shared.h"
 
-INCLUDE_ASM("weapon/nonmatchings/w_002", EntityWeaponAttack);
+extern WeaponAnimation D_12000_8017ABC4[];
```

Comment:
I considered naming these symbols (something like `WeaponAttackAnimation`). They are private to each weapon and only used by this function afaict, so while a common name doesn't help with reuse, it's nicer to read than an address label. I'm open to comments and opinions.

Reply:
I think all those data symbols used to be `static`

Reply:
Likely